docs/continuing_runs.md

```diff
@@ -0,0 +1,204 @@
+repeating the previous successful steps. For that, you should do the following:
+
+```
+haddock3 my-haddock3-config-file.cfg --restart 4
+```
+
+This operation will delete modules' folders `4_` onwards and repeat those steps
+following the information in the configuration file. You must run the `haddock3`
+command from within the same directory as you did for the original run.
+
+# Expanding a previous run from a given step
+
+The `--restart` functionality explained above allows some exciting variations.
+We will see here how to expand a previous run with additional modules.
+
+Run 1 completed successfully, but now we want to complement our run with a
+`mdref` step and perform a final CAPRI score evaluation. For that, add the
+parameters for the new modules to the end of the original configuration file.
+You must keep the rest of the configuration file the same.
+
+Navigate back to the same folder you executed the previous runs and:
+
+```
+haddock3 expanded-configuration-file.cfg --restart 6
+```
+
+In this case, the `--restart` option takes the value `6` because we want to
+continue the run from step 6, the new `mdref` step.
+
+# Editing and expanding a previous run from a given step
+
+Following the same rationale from the previous example, we can edit and expand a
+previous run. For that, edit the original configuration file by altering
+(add/edit/remove) the information for the new modules you want to execute. For
+example, delete steps `4_` onwards and perform a `mdref` only. You should delete
+the parameters for those modules in the original configuration file and replace
+them by the parameters for a `mdref` execution. In the newly edited
+configuration file, the`mdref` is now the `4_` module, so you should restart
+from four.
+
+```
+haddock3 edit-config-file.cfg --restart 4
+```
+
+This setup would result in:
+
+```
+0_topoaa/
+1_rigidbody/
+2_caprieval/
+3_seletop/
+4_mdref
+data
+log
+```
+
+As you can see, we have reused the first four modules, up to `3_seletop`, and
+replaced the `4_flexref` and `5_caprieval` with a `4_mdref` module. You could
+execute as many modules as you wish after `4_mdref`.
+
+# Copy a run to a new folder
+
+Importantly to know, you can execute all previous examples on copies of the
+original run. There is no need to rewrite the initial run if you wish to keep
+it. For that, copy the original run to a new folder:
+
+```
+cp -r run1 run2
+```
+
+In the configuration file, edit the `run_dir` parameter to `run2` so it
+matches with the newly created folder. Then, proceed by editing the
+configuration file as discussed in the previous examples. Finally,
+
+```
+haddock3 new-config-file-run2.cfg --restart N
+```
+
+where `N` is the step you wish to start from, as explained previously. Notably,
+you do not need to copy the whole folder when copying the original run. Instead,
+you can copy only the modules folders you wish to maintain plus the `data`
+folder.
+
+# Extend an unknown run
```

Comment:
Again unknown in this case is not the proper naming. But this use case should be useful indeed.

Ideally if you want to only run caprieval, you might want to run it for various stages of the run... i.e. insert it between existing steps.

Reply:
Therefore, we need the capacity to add intermediate **analysis** stages. Which would require the renaming of the consequent folders. Sounds good?

Reply:
Yes - but in that case a restart should not delete existing steps, only add the missing ones...

Reply:
Agree. Likely we would need another flag different from `--restart`.